azurerm/resource_arm_virtual_network_gateway_connection.go

```diff
@@ -79,13 +79,19 @@ func resourceArmVirtualNetworkGatewayConnection() *schema.Resource {
 			"enable_bgp": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Default:  false,
+				Computed: true,
+			},
+
+			"use_policy_based_traffic_selectors": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
 			},
 
 			"routing_weight": {
 				Type:     schema.TypeInt,
 				Optional: true,
-				Default:  10,
+				Computed: true,
```

Comment:
is this same default returned from Azure?

Reply:
Same reasoning as above, 10 is the default value, but it should not be fixed in the provider.